application-accelerator/provenance.hbs.md

```diff
@@ -21,3 +21,54 @@ project:
 
 For more information about the structure of the file and how to enable application bootstrapping
 provenance, see [Provenance transform](creating-accelerators/transforms/provenance.hbs.md).
+
+## <a id="amr"></a> Integration with AMR
+
+Application Accelerator integrates with [Artifact Metadata Repository](../scst-store/overview.hbs.md) (AMR).
+
+When you generate an application with an accelerator, an event that contains the same information captured by the `Provenance` transform is sent to the AMR store.
+
+The relevant data saved to AMR are:
+
+### <a id='appacceleratorruns'></a> AppAcceleratorRuns (alpha)
+
+The `AppAcceleratorRuns` data model represents new projects running from Git
+repositories. An `accelerator.yaml` file in the repository declares input
```

Comment:
>An `accelerator.yaml` file in the repository declares input
>options for the accelerator. This file contains instructions for processing the
>files when you generate a new project. 

This is somewhat true, but kind of irrelevant here.

>Observer sends this information to the
> CloudEvent Handler to store `AppAcceleratorRuns`.

This sentence is erroneous. "Observer" (if it relates to what I think it does) is not involved here.

Reply:
I believe Observer in this context is the AMR Observer - and yes, if I remember correctly App Accelerator sends data directly to the CloudEvent Handler? 

So we can reword to 
"This information is sent to the CloudEvent Handler to store `AppAcceleratorRuns`." ?

Reply:
yes